creatorkit/Glossary.md

```diff
@@ -0,0 +1,17 @@
+# Glossary
+
+**Creator Kit** - This is the name of the kit [available here](http://www.imgtec.com) that includes the hardware you need to complete the projects detailed in this guide.
+
+**Ci40** - Ci40 is largest board in your kit and is the IoT hub. It connects to the Internet, and has many connectivity options for constrained devices to connect to (WiFi/6LowPAN/BlueTooth). See full details of Ci40 on the [FlowM2M website](http://www.flowcloud.io)
+
+**IoT** - Abbreviation of "Internet of Things". IoT is a concept in which many devices are connected together to work more efficiently/effectively. Any system in which multiple devices are connected together and connected to the Internet can be considered to be an IoT system.
+
+**Contrained Device** - This term is a catch-all for very simple devices that connect to your Ci40 in a local network. Your Clicker boards included in the kit are an example of a Constrained Device.
+
+**OpenWRT** - This is the operating system used on Ci40 for the Creator Kit projects. Ci40 can run a variety of operating systems, but OpenWRT is used in these examples. [Openwrt website](https://openwrt.org/)
+
+**6LowPan Clicker** - 2 6LowPAN Clicker boards are included in the Creator Kit to allow you to complete hte 5 projects. Each Clicker connects to the Ci40 over 6LowPAN and acts as a remote sensor/actuator by using Click boards
+
+**Click** - 3 Click boards are included in your kit (Thermo3, Relay, PIR Motion Sensor) and many more are available from [MikroElektronika](http://www.mikroe.com/click/). These boards can be used to add new functions to your Clicker and Ci40 boards.
+
+**6LowPAN** - A low power WiFi mechanism ideal for connecting small remote devices.
```

Comment:
Lets add the definition first, then describe in simple words.
6LoWPAN is an acronym of IPv6 over Low power Wireless Personal Area Networks